railties/lib/rails/application/finisher.rb

```diff
@@ -173,6 +173,22 @@ def self.complete(_state)
         end
       end
 
+      initializer :add_builtin_route do |app|
```

Comment:
```suggestion
      initializer :add_internal_routes do |app|
```

Can we rename this without anyone relying on the initializer name? Feels like we aren't adding just one more anymore and internal would be closer than builtin.

Reply:
I've gone ahead and made this change but will leave the comment unresolved if only so others can notice this change.